app/resources/en/translation/summary.json

```diff
@@ -33,6 +33,9 @@
       },
       "fieldMinSize": {
         "summary": "Must be at least {min} characters"
+      },
+      "validationError": {
+        "summary": "There’s been a problem saving your application"
```

Comment:
Be consistent with the two errors above: `There has been a problem saving your application.`